presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1283,35 +1279,6 @@ public void testShowColumnsFromPartitions()
                 ".*Table '.*\\.tpch\\.blah\\$partitions' does not exist");
     }
 
-    @Test
-    public void testShowPartitionsFromPartitionsSystemTable()
-    {
-        String tableName = "test_show_partitions_from_partitions";
-
-        @Language("SQL") String createTable = "" +
-                "CREATE TABLE " + tableName + " " +
-                "(" +
-                "  foo VARCHAR," +
-                "  part1 BIGINT," +
-                "  part2 VARCHAR" +
-                ") " +
-                "WITH (" +
-                "partitioned_by = ARRAY[ 'part1', 'part2' ]" +
-                ") ";
-
-        assertUpdate(getSession(), createTable);
-
-        assertQueryFails(
-                getSession(),
-                "SHOW PARTITIONS FROM \"" + tableName + "$partitions\"",
-                ".*Table does not have partition columns: .*\\.tpch.test_show_partitions_from_partitions\\$partitions");
```

Comment:
Do we have a test reading from `partitioned_table$partitions$partitions`?

Reply:
Good idea, I will change this to do that.